src/MessagingThreadListItem/MessagingThreadListItem.tsx

```diff
@@ -27,7 +27,7 @@ const cssClasses = {
 export type Status = "active" | "off";
 
 type Props = {
-  children: any;
+  children?: any;
```

Comment:
Children aren't required, as potential threads (i.e. no messaging history yet, and thus no Twilio channel created) don't have any content to necessarily display here.